gma/tools/ios/testapp/testapp/game_engine.cpp

```diff
@@ -1,6 +1,6 @@
-//  Copyright © 2016 Google. All rights reserved.
```

Comment:
What even is this file?

Reply:
I'm not even sure.  I thought it was used to drive the app on iOS.  Will confirm if it's still needed.